src/app/pages/private/collection/collection.page.html

```diff
@@ -10,4 +10,11 @@
       </ion-buttons>
     </ion-toolbar>
   </ion-header>
+
+  <ion-toolbar>
+    <app-rarity-slider
+      [rarities]="rarities()"
+      [lastSelectedRarity]="lastSelectedRarity()"
```

Comment:
'selectedRarity' will be good enough for our functionality. Component probably do not care about was it rarity selected or not - he just should know which one is selected